client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -30,6 +31,23 @@ class StoreStats extends Component {
 			date: today,
 			quantity: '30'
 		};
+		const topSellersQuery = {
+			unit,
+			date: selectedDate,
+			quantity: '7',
+			limit: '3',
+		};
+		const translate = this.props.translate;
+		const moduleStrings = {
+			title: translate( 'Products', { context: 'Stats: title of module' } ),
+			item: translate( 'Title', { context: 'Stats: module row header for product title.' } ),
+			value: translate( 'Qty', { context: 'Stats: module row header for number of products sold.' } ),
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?